src/aleph/vm/utils.py

```diff
@@ -130,6 +130,14 @@ def is_command_available(command):
         return False
 
 
+def check_system_module(module_path) -> str:
+    try:
+        output = subprocess.check_output(["cat", "/sys/module", module_path], stderr=subprocess.STDOUT)
+        return str(output)
+    except subprocess.CalledProcessError:
+        return ""
```

Comment:
You don't need to use cat you can just open() and then read() it in python like a regular file

```suggestion
from pathlib import Path

def check_system_module(module_path) -> bool:
    p = Path('/sys/module/') / module_path
    if not p.exists():
        return False
    content = p.open().read().strip()
    return content == 'Y'
```

Note this return directly a bool and not a str anymore